README.md

```diff
@@ -1 +1,222 @@
-# capstonemodule1.github.io
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b> HTML + basic JavaScript Capstone MODULE 1</b></h3>
+  <h3><b>Conference Page - Creationism vs Evolution Global Summit 2023</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone: HTML + basic JavaScript Module 1 <a name="about-project"></a>
+
+**Creationism vs Evolution Global Summit 2023** This is based on an online website for a conference according to design guidelines provided by Microverse. Trought this capstone Module 1 project I show the main abilities adquired so far in the **Module 1 - The Portal**. In this Module I learned about HTML, CSS and an brief introduction to javaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">Html</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Nice background image]**
+- **[Beautifull Icons]**
+- **[Dynamic Pages]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://asadmisrani.github.io/capstonemodule1.github.io/)
+
+- This is the [Loom Walktrough Video](https://www.loom.com/share/60bcd67bfb264ba081ae8ac442537624) of my project.
```

Comment:
- [ ] As requested by the previous reviewer kindly ensure that the audio from your video recording is clear and audible and that the reviewer is able to hear you speak, highlighting a piece of code you found interesting or are proud of making.